app/styles/app.scss

```diff
@@ -161,3 +188,17 @@ h2 {
   padding-right: 18px;
   padding-left: 18px;
 }
+// need to re-name
+.col-sm-4 {
+  width: 33.33333333%;
+  color: black;
+}
+.navbar-default {
+  background-color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.